### DISCUSSION TITLE
Query on Normalized Cr value with respect to the effective area

### DISCUSSION THREAD

**Discussion by mukherjeearup** [Telecominfraproject/oopt-gnpy#485] [Q&A] [answered]
Hi GnPy Team,
Thanks for your previous help.
I have one question regarding Raman gain coeffcient used in GnPy.

In GnPy version 2.7, normalized Cr values were provided (Ref wavelength not mentioned, I assume it should be around 1455 nm):
CR_NORM = array([
    0., 7.802e-16, 2.4236e-15, 4.0504e-15, 5.6606e-15, 6.8973e-15, 7.802e-15, 8.4162e-15, 8.8727e-15, 9.2877e-15,
    1.01011e-14, 1.05244e-14, 1.13295e-14, 1.2367e-14, 1.3695e-14, 1.5023e-14, 1.64091e-14, 1.81936e-14, 2.04927e-14,
    2.28167e-14, 2.48917e-14, 2.66098e-14, 2.82615e-14, 2.98136e-14, 3.1042e-14, 3.17558e-14, 3.18803e-14, 3.17558e-14,
    3.15566e-14, 3.11748e-14, 2.94567e-14, 3.14985e-14, 2.8552e-14, 2.43439e-14, 1.67992e-14, 9.6114e-15, 7.02180e-15,
    5.9262e-15, 5.6938e-15, 7.055e-15, 7.4119e-15, 7.4783e-15, 6.7645e-15, 5.5361e-15, 3.6271e-15, 2.7224e-15,
    2.4568e-15, 2.1995e-15, 2.1331e-15, 2.3323e-15, 2.5564e-15, 3.0461e-15, 4.8555e-15, 5.5029e-15, 5.2788e-15,
    4.565e-15, 3.3698e-15, 2.2991e-15, 2.0086e-15, 1.5521e-15, 1.328e-15, 1.162e-15, 9.379e-16, 8.715e-16, 8.134e-16,
    8.134e-16, 9.379e-16, 1.3612e-15, 1.6185e-15, 1.9754e-15, 1.8758e-15, 1.6849e-15, 1.2284e-15, 9.047e-16, 8.134e-16,
    8.715e-16, 9.711e-16, 1.0375e-15, 1.0043e-15, 9.047e-16, 8.134e-16, 6.806e-16, 5.478e-16, 3.901e-16, 2.241e-16,
    1.577e-16, 9.96e-17, 3.32e-17, 1.66e-17, 8.3e-18]) # SSMF Raman coefficient profile normalized with respect to the effective area (Cr * A_eff)

Now, In GnPy version 2.8, I can see default Raman coefficient g0 at 1454 nm is provided, so I expect if I multiply this g0 values with effective area overlap function at ref freq 1454 nm, I should arrive at CR_NORM values which were provided in GnPy 2.7.

gamma_raman = g0 * effective_area_overlap(stokes_wave, raman_reference_frequency @1454 nm)

But I see, after doing this operation, "gamma_raman" values are not matching with "CR_NORM" in GnPy 2.7

gamma_raman = array([0.00000000e+00, 8.52441993e-16, 2.64356787e-15, 4.41054841e-15,
       6.15342296e-15, 7.48492470e-15, 8.45206081e-15, 9.10154932e-15,
       9.57837595e-15, 1.00086427e-14, 1.08657736e-14, 1.13007763e-14,
       1.21432386e-14, 1.32310658e-14, 1.46249010e-14, 1.60133306e-14,
       1.74581194e-14, 1.93202413e-14, 2.17203954e-14, 2.41373374e-14,
       2.62816322e-14, 2.80410200e-14, 2.97231554e-14, 3.12935353e-14,
       3.25179616e-14, 3.31988395e-14, 3.32952769e-14, 3.31315569e-14,
       3.28901385e-14, 3.24589172e-14, 3.06068428e-14, 3.26603495e-14,
       2.95741911e-14, 2.51889432e-14, 1.73456049e-14, 9.90286076e-15,
       7.21917639e-15, 6.07956599e-15, 5.82837307e-15, 7.20580801e-15,
       7.56192435e-15, 7.62115235e-15, 6.88598868e-15, 5.62918105e-15,
       3.67972760e-15, 2.75558697e-15, 2.48101339e-15, 2.21600805e-15,
       2.14406260e-15, 2.33873071e-15, 2.55731793e-15, 3.03983905e-15,
       4.83371655e-15, 5.46474318e-15, 5.22918781e-15, 4.51076853e-15,
       3.32134731e-15, 2.26025770e-15, 1.96957650e-15, 1.51798540e-15,
       1.29539886e-15, 1.13046722e-15, 9.10004391e-16, 8.43291992e-16,
       7.84922407e-16, 7.82756820e-16, 9.00051444e-16, 1.30259265e-15,
       1.54441089e-15, 1.87955941e-15, 1.77961302e-15, 1.59381599e-15,
       1.15855224e-15, 8.50704444e-16, 7.62540466e-16, 8.14510751e-16,
       9.04794469e-16, 9.63643190e-16, 9.29863390e-16, 8.34973950e-16,
       7.48290128e-16, 6.24079477e-16, 5.00652536e-16, 3.55337326e-16,
       2.03442177e-16, 1.42675226e-16, 8.98001658e-17, 2.98290682e-17,
       1.48619591e-17, 7.40448211e-18])

So, can you please explain how this g0 values are taken in GnPy 2.8 and why is this different than GnPy 2.7?

Regards,
Arup

**Comment by AndreaDAmico** [comment, marked as answer]
Dear @mukherjeearup,
Before the effective area scaling along frequency was not considered in GNPy and the CR_NORM (the mode intensity  Raman coefficient see Eq. 7 ref [1]) was erroneously obtained multiplying the Raman gain profile by the effective area of the SSMF in the C band. In this new version,  the Raman gain coefficient that comes from an actual measurement at 1454 nm (206 THz) is properly scaled using both the frequency and effective area scaling. The old value of the Raman coefficient (CR_NORM * A_eff @ 1550 nm) can be approximately recovered using Eq. 8 in ref. [1].

[1] A. D’Amico, et al. "Scalable and Disaggregated GGN Approximation Applied to a C+L+S Optical Network,"  J. Lightwave Technol. 40, 3499-3511 (2022) Section III.D
